### PR TITLE
Remove instance of 'PDS' in certain modules

### PR DESCRIPTION
**Pull request by gonzalesedwin1123**
## **Why is this change needed?**

- Remove the instances of the term "PDS" abd "Iraq" in certain OpenSPP modules

## **How was the change implemented?**

- Update all modules with the term "PDS"
- Remove unused "PDS" terms
- Modify fields or models with the appropriate names

## **New unit tests**

- None

## **Unit tests executed by the author**

- None

## **How to test manually**

- For developers only. Search for the term pds in all modules

### CODE REVIEW

**Review by emjay0921 — APPROVED** [review]

File: spp_change_request/models/mixins/source_mixin.py [hunk 1]
```diff
@@ -584,7 +583,7 @@ def _copy_service_point_ids(self, change_request_field):
                     change_request_field: rec.id,
                     "service_point_id": mrec.id,
                 }
-                self.env["pds.change.request.service.point"].create(service_points)
+                self.env["spp.change.request.service.point"].create(service_points)
```
Comment (reichie020212) [inline spp_change_request/models/mixins/source_mixin.py]: @gonzalesedwin1123 upon checking, there is no existing model with the name `spp.change.request.service.point` in OpenSPP

File: spp_change_request/models/mixins/source_mixin.py [hunk 2]
```diff
@@ -603,7 +602,7 @@ def _copy_from_group_member_ids(self, group_ref_field, group_id_field, skip_head
                         "kind_ids": kind_ids,
                         "new_relation_to_head": None,
                     }
-                    self.env["pds.change.request.src.grp"].create(group_members)
+                    self.env["spp.change.request.src.grp"].create(group_members)
```
Comment (reichie020212) [inline spp_change_request/models/mixins/source_mixin.py]: @gonzalesedwin1123 upon checking, there is no existing model with the name `spp.change.request.src.grp ` in OpenSPP

**Review by reichie020212 — CHANGES_REQUESTED** [review]

File: spp_change_request/models/mixins/source_mixin.py (hunk 1 above)
Comment (gonzalesedwin1123) [reply spp_change_request/models/mixins/source_mixin.py]: Yes but I plan to keep this function as I need to scan all that might still be using this. I will have this function removed once I am done checking everything that might be referring to this function.

File: spp_change_request/models/mixins/source_mixin.py (hunk 2 above)
Comment (gonzalesedwin1123) [reply spp_change_request/models/mixins/source_mixin.py]: Yes but I plan to keep this function as I need to scan all that might still be using this. I will have this function removed once I am done checking everything that might be referring to this function.

File: spp_change_request/models/mixins/source_mixin.py (hunk 1 above)
Comment (reichie020212) [reply spp_change_request/models/mixins/source_mixin.py]: ah, i see, I think it is better to comment it as well, then add a TODO note, something like below code

```
# TODO: Investigate this
# self.env["spp.change.request.service.point"].create(service_points)
```

File: spp_change_request/models/mixins/source_mixin.py (hunk 2 above)
Comment (reichie020212) [reply spp_change_request/models/mixins/source_mixin.py]: ahh, i see, I think it is better to comment it as well, then add a TODO note, something like below code

```
# TODO: Investigate this
# self.env["spp.change.request.src.grp"].create(service_points)
```

**Review by reichie020212 — APPROVED** [review]

**Pull request merged**